### PR TITLE
added reviews module and test #29

### PR DESCRIPTION
**Pull request by kmiddleton14**

### CODE REVIEW

File: db/models/review.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  description: {
+    type: Sequelize.STRING,
+    allowNull: true
+  },
+  rating: {
+    type: Sequelize.ENUM('1', '2', '3', '4', '5'),
+    allowNull: false
+  },
+  //TODO: authorId should be validated against userId in the user table (association methods)
+  authorId: {
```
Comment (csjiang) [inline db/models/review.js]: Should we create the foreign keys as columns in the model declarations? Won't they be automatically created when we add the associations? (e.g. we can say `Review.belongsTo(User, {as: 'author'})` -- forgive me if the syntax isn't exactly correct.)

File: db/models/review.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    allowNull: false
+  },
+  //TODO: packageId should be validated against userId in the user table (association methods)
+  packageId: {
+    type: Sequelize.INTEGER,
+    allowNull: false
+  },
+  authorName: {
+    type: Sequelize.STRING,
+    allowNull: false           //if no name is provided, should result to "anonymous" in express
```
Comment (csjiang) [inline db/models/review.js]: you can add `defaultValue: 'Anonymous'`

File: db/models/test/review.test.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+const {expect} = require('chai')
+
+describe('Review', () => {
+  before('wait for the db', () => db.didSync)
+  const exampleReview = {
+    authorName: 'Susan McGee',
+    title: 'Sun = fun',
+    description: "The sun that I ordered was quite nice, round, and pleasant. and no i'm not talking about my husband",
+    rating: '5',
+    authorId: 3,
```
Comment (csjiang) [inline db/models/test/review.test.js]: Having the foreign key fields here may break your tests once you have removed the column declarations from the review.js model file, but once we add in the associations the tests should pass... I think.

**Comment by csjiang** [comment]
Make sure to add tests for your rating calculation class method.

**Review by csjiang — CHANGES_REQUESTED** [review]

**Comment by kmiddleton14** [comment]
We updated our module in the review.js so that our class method is returning the packages by the same ID

**Review by csjiang — APPROVED** [review]

**Pull request merged**